examples/example-gateway/idl/endpoints/bar/bar.thrift

```diff
@@ -35,7 +35,10 @@ struct QueryParamsOptsStruct {
 }
 
 struct ParamsStruct {
-    1: required string userUUID (zanzibar.http.ref = "params.user-uuid")
+    1: required string userUUID (
+        zanzibar.http.ref = "params.user-uuid"
+        go.tag = "json:\"-\""
```

Comment:
Does this remove the field from the json body?

Reply:
yes it does.